src/resolvers/__tests__/set-light-attributes.test.js

```diff
@@ -0,0 +1,33 @@
+import { query, bridge, createLight } from '../../test-utils';
+
+jest.mock('../../../bridge.json');
+
+describe('Light attributes mutation', () => {
+  let get, put, spy, light;
+
+  beforeEach(() => {
+    light = createLight({ on: true });
+    spy = jest.fn(() => ({ fake: true }));
+
+    put = bridge.put('/lights/10').reply(200, spy);
+    get = bridge.get('/lights/10').reply(200, light);
+  });
+
+  afterEach(() => {
+    get.done();
+    put.done();
+  });
+
+  it('sets the light attributes', async () => {
+    await query`mutation {
+      hue {
+        setLightAttributes(id: 10 attributes: { name: "foo" }) { name }
```

Comment:
I've seen a pattern in other GraphQL servers which is growing on me. Mutations follow the naming convention `update<Type>`/`create<Type>`/`delete<Type>`. Queries are just the lowercased type name, like `light(...)`, while mutations would be `createLight`, `updateLight`, and `deleteLight`. Keeps the API consistent and easy to guess at.

Not asking for any changes, just thinking out loud :wink:

Reply:
Yeah, that makes sense. I read [this article](https://dev-blog.apollodata.com/designing-graphql-mutations-e09de826ed97) recently which may also be relevant 🙂

Reply:
Nice find! The author has some good mind thoughts. I like them 👌

Reply:
I'd be down with refactoring to this convention after the PR 🙂